### PR TITLE
Ls 71

### PR DESCRIPTION
**Pull request by nitro-marky**
GET capacity/{capacity_id}/creation/{creation_id} with happy and sad test.

### CODE REVIEW

File: test/integration/capacity.test.ts [hunk 1]
```diff
@@ -90,8 +90,17 @@ describe('capacity', () => {
...
+      identitySelfMock()
+      const response = await get(app, `/capacity/${seededCapacityId}/creation/${seededTransactionId}`)
+      expect(response.status).to.equal(200)
+    })
+
   })
 
+
+
   describe('sad path', () => {
```
Comment (n3op2) [inline test/integration/capacity.test.ts:104]: 'sad path' - :))))

**Review by n3op2 — APPROVED** [review]

File: src/lib/db/index.ts [hunk 2]
```diff
@@ -85,6 +85,10 @@ export default class Database {
     return this.db().transaction().select('*').where({ id })
   }
 
+  getTransactions = async () => {
+    return this.db().transaction().select('*')
+  }
+
```
Comment (jonmattgray) [inline src/lib/db/index.ts:91]: function isn't used anywhere?

File: test/integration/capacity.test.ts [hunk 3]
```diff
@@ -90,8 +90,17 @@ describe('capacity', () => {
       expect(capacity.latestTokenId).to.equal(mockTokenId)
       expect(capacity.originalTokenId).to.equal(mockTokenId)
     })
+
+    it('it should get transactions', async () => {
+      identitySelfMock()
```
Comment (jonmattgray) [inline test/integration/capacity.test.ts:95]: shouldn't be required as requesting a transaction doesn't hit the identity service

File: test/integration/capacity.test.ts [hunk 4]
```diff
@@ -120,5 +129,10 @@ describe('capacity', () => {
       const response = await post(app, `/capacity/${seededCapacityId}/creation`, {})
       expect(response.status).to.equal(500)
     })
+
+    it('non-existant Creation ID', async () => {
```
Comment (jonmattgray) [inline test/integration/capacity.test.ts:133]: there could be another sad path test for the other 404 error the route throws

File: test/integration/capacity.test.ts [hunk 5]
```diff
@@ -90,8 +90,17 @@ describe('capacity', () => {
       expect(capacity.latestTokenId).to.equal(mockTokenId)
       expect(capacity.originalTokenId).to.equal(mockTokenId)
     })
+
+    it('it should get transactions', async () => {
+      identitySelfMock()
+      const response = await get(app, `/capacity/${seededCapacityId}/creation/${seededTransactionId}`)
```
Comment (jonmattgray) [inline test/integration/capacity.test.ts:96]: good idea to assert the contents of `response.body` too to make sure it's returning what we want

Comment (n3op2) [reply test/integration/capacity.test.ts:96]: yea, I would say even good to assert all properties so then we make a change test will flag up.

File: test/integration/capacity.test.ts (hunk 3 above)
Comment (n3op2) [reply test/integration/capacity.test.ts:95]: this is for phuture

Comment (jonmattgray) [reply test/integration/capacity.test.ts:95]: what?

Comment (jonmattgray) [reply test/integration/capacity.test.ts:95]: what is phuture?

File: test/seeds/index.ts [hunk 6]
```diff
@@ -39,6 +41,16 @@ export const seed = async () => {
     },
   ])
 
+  await db.transaction().insert([
+    {
+      id: seededTransactionId,
+      token_type: 'DEMAND',
```
Comment (jonmattgray) [inline test/seeds/index.ts:47]: you can import the `TokenType` enum and use that instead of writing out the string, same with `state`

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**